cli/internal/runcache/runcache.go

```diff
@@ -59,77 +52,57 @@ func AddFlags(opts *Opts, flags *pflag.FlagSet) {
 	}
 }
 
-var _outputModeHelp = `Set type of process output logging. Use full to show
```

Comment:
I moved this into the flag definition because it didn't feel like having it as a separate variable added anything. Was that correct or is there some benefit I missed here?